Frontend/ui-customer/src/Helpers/Utils.purs

```diff
@@ -910,3 +907,42 @@ findSpecialPickupZone stringGeoJson gates lat lon =
                 Nothing -> Nothing
         Nothing -> Nothing
     _, _ -> Nothing
+
+newtype CityMetroConfig = CityMetroConfig { 
+    logoImage :: String
+  , title :: String
+  , mapImage :: String
+  , bannerImage :: String
+  , bannerBackgroundColor :: String
+  , bannerTextColor :: String
+  , termsAndConditionsUrl :: String
+  , termsAndConditions :: Array String
+  , errorPopupTitle :: String
+  , operationalHours :: 
+    { startTime :: String
+    , endTime :: String
+    }
+  , showCancelButton :: Boolean
+}
+
+getMetroConfigFromCity :: City -> CityMetroConfig
```

Comment:
@VD321 Where do we use Metro times from this config?

Reply:
Shifted to v1-configurations